File: c/ProjectEuler/1_.c
#include<stdio.h>

long fib(long n){
    if (n==1){
        return 0;
    }

    else if (n==2){
        return 1;
    }
    else{
        return fib(n-1)+fib(n-2);
    }
}  

int main(){
    printf("%ld\n",fib(5));
    long i=2;
    long sum;
    // int arr[];
    while (i<20){
        long num=fib(i);
        if (num%2==0){
            sum=sum+num;
            printf("%ld",i);
            i=i+1;
        }
        printf("%ld",i);
        // i=+1;

    }
    printf(" the sum is %ld\n",sum);
    // printf("%d\n",fib(5));
}

File: Python/SlidingWindow/1_.py


def slidingWindow(arr,k):
    n=len(arr)
    if(n<k):
        return -1
    sum_=sum(arr[:k])
    MAX=sum_
    
    for i in range(n-k):
        sum_=sum_-arr[i]+arr[i+k]
        MAX=max(MAX,sum_)
    
    return MAX

arr = [1, 4, 2, 10, 2, 3, 1, 0, 20]
k = 3
print(slidingWindow(arr,k))

    
    




